### PR TITLE
feat: Magically move your site to FC from the dashboard

### PR DESCRIPTION
**Pull request by gavindsouza**
Screenshot 2020-08-13 at 1 59 44 PM

Depends on https://github.com/frappe/agent/pull/12

### CODE REVIEW

File: press/utils.py [hunk 1]
```diff
@@ -106,3 +106,56 @@ def get_minified_script():
 	migration_script = "../apps/press/press/scripts/migrate.py"
 	script_contents = open(migration_script).read()
 	return minify(script_contents)
+
+
+def verify_frappe_site(site):
```
Comment (adityahase) [inline press/utils.py]: Verify with credentials.

File: press/utils.py [hunk 2]
```diff
@@ -106,3 +106,56 @@ def get_minified_script():
...
+	api = api_key and api_secret
+	passwd = usr and pwd
+
+	def url(path):
+		host = site.split(":")[0]
+		file = path.lstrip(f"./{host}/private/")
+		url = f"{schema}://{site}/{file}?sid={sid}"
+		return url
+
+	if api:
```
Comment (adityahase) [inline press/utils.py]: Get rid of this. Setting up a token, just for this is too much work anyway.

File: press/utils.py [hunk 3]
```diff
@@ -106,3 +106,56 @@ def get_minified_script():
...
+		data = res.json()
+		if data.get("message") == "pong":
+			status = True
+	except Exception:
+		pass
+
+	return {"schema": schema, "status": status}
+
+
+def get_frappe_backups(site, auth):
```
Comment (adityahase) [inline press/utils.py]: Can we move this to client-side? So local sites would also work?

File: dashboard/src/views/NewSite.vue [hunk 4]
```diff
@@ -150,6 +150,120 @@
...
+										class="w-5 h-5 p-1 mr-2 text-green-500 bg-green-100 rounded-full"
+										v-if="verifiedFrappeSite === true"
+									/>
+									<FeatherIcon
+										name="x"
+										class="w-5 h-5 p-1 mr-2 text-red-500 bg-red-100 rounded-full"
+										v-if="verifiedFrappeSite === false"
+									/>
+								</p>
+								<Button class="mt-6 w-30" type="primary" @click="verifySite">
```
Comment (adityahase) [inline dashboard/src/views/NewSite.vue]: Too many visible primary buttons at once.

File: press/utils.py (hunk 3 above)
Comment (gavindsouza) [reply press/utils.py]: we're passing the file URL directly to agent, local sites wouldn't work anyway

**Pull request merged**